libs/coin-modules/coin-tron/src/logic/listOperations.ts

```diff
@@ -1,5 +1,10 @@
 import { Operation } from "@ledgerhq/coin-framework/api/index";
+import { fetchTronAccountTxs } from "../network";
+import { fromTrongridTxInfoToOperation } from "../network/trongrid/trongrid-adapters";
 
-export function listOperations(_address: string): Promise<[Operation[], string]> {
-  throw Error("Not implemented yet");
+export async function listOperations(_address: string): Promise<[Operation[], string]> {
```

Comment:
why `_address` and not just `address` ?

Reply:
I can change it, it's what was on develop